app/src/main/java/com/example/ithaca_transit_android_v2/Models/MapState.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.ithaca_transit_android_v2.Models
+
+import com.example.ithaca_transit_android_v2.models.Coordinate
+import com.example.ithaca_transit_android_v2.models.Location
+import java.util.*
+
+sealed class MapState(
+    val trip:Trip = Trip(
+        emptyList(),
+        Location("", Coordinate(0.0, 0.0), ""),
+        Location("", Coordinate(0.0, 0.0), ""),
+        false,
+        Date(0),
+        Date(0),
+        0)
+) {}
+
+class LaunchState():MapState()
```

Comment:
think of `MapState` as a parent class. if `MapState` has `Trip` as a param, then all its children need to have `Trip` as a param as well!

Reply:
should be `class LaunchState(trip : Trip) : MapState(trip)`